pkg/sql/show_fingerprints.go

```diff
@@ -71,8 +72,22 @@ type showFingerprintsNode struct {
 func (p *planner) ShowFingerprints(
 	ctx context.Context, n *tree.ShowFingerprints,
 ) (planNode, error) {
+
+	op := "SHOW EXPERIMENTAL_FINGERPRINTS"
+	evalOptions, err := evalShowFingerprintOptions(ctx, n.Options, p.EvalContext(), p.SemaCtx(),
+		op, p.ExprEvaluator(op))
+	if err != nil {
+		return nil, err
+	}
+
 	if n.TenantSpec != nil {
-		return p.planShowTenantFingerprint(ctx, n.TenantSpec, n.Options)
```

Comment:
nb: a tenant level fingerprint cannot use EXCLUDE COLUMNS. We should error if the user provides that option here as well.

Reply:
Done